x-pack/plugins/task_manager/server/task_manager.ts

```diff
@@ -262,6 +289,20 @@ export class TaskManager {
     } catch (e) {
       this.logger.error('Could not sanitize task definitions');
     }
+
+    Object.values(taskDefinitions)
+      .filter(taskDefinition => taskDefinition.hasOwnProperty('maxConcurrency') && taskDefinition.maxConcurrency! > 0)
+      .forEach(taskDefinition => {
+        console.log(`CREATING POLLER FOR ${taskDefinition.type}`)
```

Comment:
Does this actually create a different poller for each task type?

Reply:
Just saw your question :)
Yes, it does.

Reply:
It uses the same pool, capacity etc. but different pollers, so the `max_docs` can be adjusted between pollers.